Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apt-get install -y nodejs
 
 RUN npm install
 RUN npm install babel-core@^7.0.0-bridge.0 --save-dev --dev jest --save-dev vue-jest vue-template-compiler --save-dev jest-serializer-vue -g yarn
-RUN npm run dev
+#RUN npm run dev
```

Comment:
This was added to get Heroku to build the static assets.  We have to confirm with a push to Heroku where the environment is set to 'production' to test if this still builds and whitenoise serves the file.